modules/vactory_cross_content/src/Field/InternalVCCFieldItemList.php

```diff
@@ -75,7 +92,12 @@ protected function computeValue() {
             $value['nodes'] = [];
             /** @var NodeInterface $node */
             $entity_repository = \Drupal::service('entity.repository');
```

Comment:
@khalidbouhouch77 
Use of dependency injection for `\Drupal::service('entity.repository')`